lib/aggregate/aggregate_store.rb

```diff
@@ -285,5 +285,13 @@ def ensure_saved_changes_up_to_date
         save_changes
       end
     end
+
+    def store_value?(aggregate_attribute, value)
+      skip_on_default =
+        if aggregate_attribute.respond_to?(:skip_default?)
```

Comment:
You can add this to the base class and then we don't need to check respond_to?  It would be clearer that way as well.